openapi.yml

```diff
@@ -0,0 +1,506 @@
+openapi: 3.0.0
+info:
+  description: Workflow Server HTTP API
+  version: 1.0.0
+  title: Workflow Server HTTP API
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://workflow-service-prod.stanford.edu'
+    description: Production service
+    variables:
+      env:
+        default: prod
+  - url: 'https://workflow-service-stage.stanford.edu'
+    description: Staging service
+    variables:
+      env:
+        default: stage
+paths:
+  /{repo}/objects/{druid}/lifecycle:
+    get:
+      summary: Retrieve workflow lifecycle for an object
+      responses:
+        '200':
+          description: Workflow lifecycle found
+          content:
+            application/xml:
+              schema:
+                type: string
```

Comment:
I'm not saying you *need* to do this here, but OpenAPI does provide a rich way to validate XML documents. See https://github.com/sul-dlss/preservation_catalog/blob/master/openapi.yml#L289-L300 Essentially, treat them as though they're JSON: https://github.com/sul-dlss/preservation_catalog/blob/master/openapi.yml#L399-L474 (a long and gnarly one, but maybe your XML here is less of a nightmare)

Reply:
Yeah - I'm leaning toward doing response validation in it's own PR as I'm guessing there will be a lot of broken tests (i'll double check that) and it will clutter this PR.

Reply:
ok! resolving